src/azure-cli/azure/cli/command_modules/cosmosdb/commands.py

```diff
@@ -42,6 +43,114 @@ def load_command_table(self, _):
         g.custom_command('update', 'cli_cosmosdb_update')
         g.custom_command('list', 'cli_cosmosdb_list')
 
+    # SQL api
+    with self.command_group('cosmosdb sql', is_preview=True):
+        pass
+    with self.command_group('cosmosdb sql database', cosmosdb_sdk, client_factory=cf_db_accounts) as g:
+        g.custom_command('create', 'cli_cosmosdb_sql_database_create')
+        g.command('list', 'list_sql_databases')
+        g.command('show', 'get_sql_database')
+        g.command('delete', 'delete_sql_database')
```

Comment:
For delete commands, would you like to add `confirmation=True` to allow prompt of delete confirmation. `--yes/-y` will be added automatically to run without prompting.